dockernet/config.sh

```diff
@@ -28,9 +31,9 @@ HOST_CHAINS=()
 #  - STARS
 #  - HOST (Stride chain enabled as a host zone)
 if [[ "${ALL_HOST_CHAINS:-false}" == "true" ]]; then 
-  HOST_CHAINS=(GAIA OSMO HOST)
+  HOST_CHAINS=(GAIA OSMO JUNO UMEE)
 elif [[ "${#HOST_CHAINS[@]}" == "0" ]]; then 
-  HOST_CHAINS=(GAIA)
+  HOST_CHAINS=(GAIA UMEE)
```

Comment:
for the sake of debugging, might be easier to run with just UMEE at the moment
```suggestion
  HOST_CHAINS=(UMEE)
```

Reply:
Done !